scripts/set-codecommit-cross-account-access/on-start.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+# OVERVIEW
+# This script sets cross-account CodeCommit access, so you can work on repositories hosted in another account.
+# You'll need to create a role in AccountA granting repositories access to AccountB as instructed here:
+# https://docs.aws.amazon.com/sagemaker/latest/dg/nbi-git-cross.html
+# More information about the credential helper here:
+# https://docs.aws.amazon.com/codecommit/latest/userguide/setting-up-https-unixes.html#setting-up-https-unixes-credential-helper
+
+# PARAMETERS
+ROLE_ARN=arn:aws:iam::YourAccount:role/YourCrossAccountRepoAccess-Role
```

Comment:
Could we follow the terminology in the official documentation to disambiguate the account that contains the CodeCommit repository and the one that contains the Notebook Instance? 

This will become


```suggestion
ROLE_ARN=arn:aws:iam::CodeCommitAccount:role/CrossAccountRepositoryContributorRole
```